### PR TITLE
Do not render image height

### PR DESCRIPTION
**Pull request by mariobodemann**
Do respect the aspect ratio of the incoming images, by not scaling them in height.

### CODE REVIEW

File: views/mixins.jade [hunk 1]
```diff
@@ -16,11 +16,11 @@ mixin renderImage(image, slug, w)
   if image && image.fields.file
     if slug != ""
       a(href="/products/"+slug data-nav) 
-        img(src=image.fields.file.url alt=image.fields.title width=w)
+        img(src=image.fields.file.url+"?w="+w alt=image.fields.title width=w)
```
Comment (Khaledgarbaya) [inline views/mixins.jade]: Can you remove the width=w since the server will return the desired width already

Comment (mariobodemann) [inline views/mixins.jade]: Hmm,

http://webmasters.stackexchange.com/questions/68491/specifying-image-dimensions-to-improve-browser-performance

Maybe the width argument should still stay?

**Comment by Khaledgarbaya** [comment]
LGTM :shipit:

**Pull request merged**